qbft/message_container.go

```diff
@@ -3,82 +3,98 @@ package qbft
 import (
 	"bytes"
 	"encoding/json"
+
 	"github.com/bloxapp/ssv-spec/types"
 )
 
 type MsgContainer struct {
-	Msgs map[Round][]*SignedMessage
+	Msgs map[Round][]*types.SignedSSVMessage
 }
 
 func NewMsgContainer() *MsgContainer {
 	return &MsgContainer{
-		Msgs: map[Round][]*SignedMessage{},
+		Msgs: map[Round][]*types.SignedSSVMessage{},
 	}
 }
 
 // AllMessaged returns all messages
-func (c *MsgContainer) AllMessaged() []*SignedMessage {
-	ret := make([]*SignedMessage, 0)
+func (c *MsgContainer) AllMessaged() []*types.SignedSSVMessage {
```

Comment:
unrelated to this PR, but if we already changed this line we can finally fix this typo 😅 (`AllMessages`)

Reply:
Oh man, definitely